src/collective/lazysizes/profiles/base/metadata.xml

```diff
@@ -1,4 +1,4 @@
 <?xml version="1.0"?>
 <metadata>
-  <version>7</version>
```

Comment:
why do we need to change the profile?

Reply:
because the profile has changed and should be re-loaded.

on the other hand - for plone 4 nothing is changed...